Source/Core/Core/HW/WiimoteEmu/Extension/BalanceBoard.h

```diff
@@ -0,0 +1,75 @@
+// Copyright 2010 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <array>
+
+#include "Core/HW/WiimoteCommon/WiimoteReport.h"
+#include "Core/HW/WiimoteEmu/Dynamics.h"
+#include "Core/HW/WiimoteEmu/Extension/Extension.h"
+
+namespace ControllerEmu
+{
+class AnalogStick;
+class Triggers;
+}  // namespace ControllerEmu
+
+namespace WiimoteEmu
+{
+enum class BalanceBoardGroup
+{
+  Balance,
+  Weight
+};
+
+class BalanceBoard : public Extension1stParty
+{
+public:
+  union DataFormat
```

Comment:
This can just be turned into a struct the current way this is.

Reply:
It was declared the same way for Nunchuk (which I based that code off of):

https://github.com/dolphin-emu/dolphin/blob/a9e04e0a72e59ee0cb9e0daf017561b2fb4b93dc/Source/Core/Core/HW/WiimoteEmu/Extension/Nunchuk.h#L53-L69

Is there a reason why that one was done that way, or is it just because `ButtonFormat` is an actual union?

Reply:
This is equivalent to just declaring a struct. There's no other element in the union to share the same memory location as the unnamed struct. Nunchuk's union should also likely be converted into a struct as well